src/library/trackset/baseplaylistfeature.h

```diff
@@ -93,6 +93,7 @@ class BasePlaylistFeature : public BaseTrackSetFeature {
     // Get the QModelIndex of a playlist based on its id.  Returns QModelIndex()
     // on failure.
     QModelIndex indexFromPlaylistId(int playlistId);
+    void exportPlaylistById(int playlistId);
```

Comment:
Would you say this is a good spot to place function? Is this a clear function name?